scripts/tpg_find_overactive_channels.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python
+"""
+For a given TPStream, plot the channel histogram and print
+the channels that are above a given limit and Tuckey's fence.
+"""
+
+
+from trgtools import TPReader
+
+import click
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+import os
+
+
+# As of 05-13-2024.
+DETECTOR_CHANNELMAPS = {
+            "HD": os.path.join(os.environ["DETCHANNELMAPS_SHARE"], "config/pd2hd/PD2HDChannelMap_v6.txt"),
+            "VD": os.path.join(os.environ["DETCHANNELMAPS_SHARE"], "config/pd2vd/PD2VDBottomTPCChannelMap_v1.txt"),
+            "VDCB": os.path.join(os.environ["DETCHANNELMAPS_SHARE"], "config/vdcoldbox/vdcbce_chanmap_v4.txt")
```

Comment:
Out of curiosity, do you happen to know what happened to the hdcoldbox channel maps? I don't see them in `$DETCHANNELMAPS_SHARE/config`, and we do have some old hd tpstreams/raw data saved.

Reply:
I am uncertain. I did see remnants that referred to the HD cold box maps, but I've never come across it in `detchannelmaps`.